types/d3-geo/index.d.ts

```diff
@@ -2,7 +2,7 @@
 // Project: https://github.com/d3/d3-geo/
 // Definitions by: Hugues Stefanski <https://github.com/Ledragon>, Tom Wanzek <https://github.com/tomwanzek>, Alex Ford <https://github.com/gustavderdrache>, Boris Yankov <https://github.com/borisyankov>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.4
```

Comment:
Can you already set this back to 2.3 (we should not need a TS 2.4 minimum constraint at this point, unless you were thinking of using something like `string enums`)

Setting the constraint back to TS 2.3 will remove the Travis/Test conflicts with `d3`.